OrthoEvol/Tools/ftp/ncbiftp.py

```diff
@@ -26,18 +26,27 @@ class NcbiFTPClient(BaseFTPClient):
     """Access NCBI's FTP servers with ease."""
 
     def __init__(self, email):
+        """Initialize the NcbiFTPClient.
+
+        :param email: A valid email address to use a password.
+        :type email: str
+        """
         _ncbi = 'ftp.ncbi.nlm.nih.gov'
-        super().__init__(_ncbi, email, keepalive=False, debug_lvl=0)
-        self._datafmt = '%m-%d-%Y@%I:%M:%S-%p'
-        self._date = str(datetime.now().strftime(self._datafmt))
+        super().__init__(_ncbi, user="anonymous", password=email)
```

Comment:
Passively give the user access to the other parameters in the BaseFTP class using kwargs, becasue use-case can always vary.
```suggestion
        super().__init__(_ncbi, user="anonymous", password=email, **kwargs)
```